### PR TITLE
New Sample: ProxyTool

### PR DESCRIPTION
**Pull request by areino**
# New Sample: ProxyTool
Adding sample using RTR API to centrally change proxy settings across host group or CID. 

- [x] Documentation
- [x] Code sample

#### Unit test coverage
```shell
COVERAGE TESTING NOT REQUIRED FOR SAMPLES
```

#### Bandit analysis
```shell
[main]	INFO	running on Python 3.9.9

Run started:2022-10-27 17:55:58.141279

Test results:
	No issues identified.

Code scanned:
	Total lines of code: 128
	Total lines skipped (#nosec): 0

Run metrics:
	Total issues (by severity):
		Undefined: 0
		Low: 0
		Medium: 0
		High: 0
	Total issues (by confidence):
		Undefined: 0
		Low: 0
		Medium: 0
		High: 0
Files skipped (0):
```

## Added features and functionality
+ New Sample: ProxyTool v3.3 - Update CrowdStrike sensor proxy configurations on Windows remotely.
    - `samples/proxytool/proxytool_3.3.py`

### CODE REVIEW

File: samples/proxytool/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+![CrowdStrike Falcon](https://raw.githubusercontent.com/CrowdStrike/falconpy/main/docs/asset/cs-logo.png)
+![Twitter URL](https://img.shields.io/twitter/url?label=Follow%20%40CrowdStrike&style=social&url=https%3A%2F%2Ftwitter.com%2FCrowdStrike)
+
+# ProxyTool v3
+This example focuses on leveraging CrowdStrike's Hosts, Host Groups, and Real-Rime Response API.
+
+It is a script that fetches CID or Host Group hosts, and uses the batch command and offline queuing of Real-Tike Response API to centrally
```
Comment (jshcodes) [inline samples/proxytool/README.md:7]: Real Time

File: samples/proxytool/proxytool-3.3.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    else:
+        exit()
+
+
+    ## Commands to change proxy config
+    ## Delete DisableProxy, PAC, PN, PP in both locations. Change CsProxyHostname and CsProxyport with new values
+
+
+    registry_stores = ["HKLM:\SYSTEM\Crowdstrike\{9b03c1d9-3138-44ed-9fae-d9f4c034b88d}\{16e0423f-7058-48c9-a204-725362b67639}\Default", 
+                        "HKLM:\SYSTEM\CurrentControlSet\Services\CSAgent\Sim"]
```
Comment (jshcodes) [inline samples/proxytool/proxytool-3.3.py:162]: These should be defined as raw strings due to the formatting character collisions.

```python
    registry_stores = [r"HKLM:\SYSTEM\Crowdstrike\{9b03c1d9-3138-44ed-9fae-d9f4c034b88d}\{16e0423f-7058-48c9-a204-725362b67639}\Default", 
                        r"HKLM:\SYSTEM\CurrentControlSet\Services\CSAgent\Sim"]
```

File: samples/proxytool/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+It is a script that fetches CID or Host Group hosts, and uses the batch command and offline queuing of Real-Tike Response API to centrally 
+and conveniently issue Falcon sensor proxy configuration changes.
+
+- It uses native RTR commands, which will not trigger a detection/prevention in relation to sensor anti-tampering. 
+- Because it uses the RTR API it is run centrally through our cloud, it does NOT need to be distributed to each targeted host. 
+- The script uses the queuing feature of RTR, so hosts don't need to be online at the time the script is executed, they will receive the commands if they connect to our cloud within the next 7 days. 
+
+
+> __WARNING: This script has the potential to disrupt the communications between the Falcon sensor and the cloud. It is suggested to test with a limited Host Group first to troubleshoot any issues.__
+
```
Comment (jshcodes) [inline samples/proxytool/README.md:16]: > __:bangbang:WARNING:bangbang:__ 
> This script has the potential to disrupt communications between the Falcon sensor and the cloud. It is recommended users test with a limited Host Group first to troubleshoot any issues.

**Review by jshcodes — CHANGES_REQUESTED** [review]
Excited to see the new sample!  Couple of very minor tweaks.

**Review by areino — COMMENTED** [review]
Changed as suggested

**Review by jshcodes — APPROVED** [review]
Thank you for your contribution!! 😄

**Pull request merged**